File: src/main/java/com/blockout/workersearch/models/dto/team/Team.java
package com.blockout.workersearch.models.dto.team;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Team {
    private Long id;
    private String clubId;
    private String name;
    private String shortName;
    private Boolean active;
    private LocalDateTime lastUpdate;
    private String leagueCode;
    private String divisionName;
    private TeamFormat format;
    private TeamGender gender;
    private Long followersCount;
}

File: apps/backend/club-scraper/models/team.py
from dataclasses import dataclass
from datetime import datetime
from typing import Optional

@dataclass
class Team:
    clubId: str
    name: str
    shortName: str
    leagueCode: str
    divisionId: str
    id: Optional[int] = None
    gender: Optional[str] = None
    format: Optional[str] = None
    active: bool = True
    createdAt: Optional[datetime] = None
    lastUpdate: Optional[datetime] = None